packages/server/utils/getTemplateScore.ts

```diff
@@ -12,8 +12,13 @@ const SATURTION = 1 // max score, we want a score that is 0-1
 const MIDPOINT = 45 // after this many days, the score will be SATURATION / 2
 const GROWTH_INTERVAL = 30 // 80% of the decline happens in this many days
 
+// Default hotness factors for ranking templates
+export const ORG_HOTNESS_FACTOR = 0.8
+export const TEAM_HOTNESS_FACTOR = 0.9
+
 const getAgeScore = (age: number) => {
-  return SATURTION / (1 + Math.exp((Math.log(81) / GROWTH_INTERVAL) * (age - MIDPOINT)))
+  const ageDays = age / 1000 / 60 / 60 / 24
```

Comment:
@mattkrick this came up when I was regression testing removing the team templates DB call - like you mentioned we'll probably change how we rank templates at some point, but I figured we should fix make this fix, since otherwise all templates will score 0 due to age being millis when the logic assumes it's days.

Reply:
sounds good!